--- llvm_release_37/lib/Support/CrashRecoveryContext.cpp
+++ macports_release_37/lib/Support/CrashRecoveryContext.cpp
@@ -332,13 +332,13 @@
 
 // FIXME: Portability.
 static void setThreadBackgroundPriority() {
-#ifdef __APPLE__
+#if defined(__APPLE__) && defined(PRIO_DARWIN_THREAD)
   setpriority(PRIO_DARWIN_THREAD, 0, PRIO_DARWIN_BG);
 #endif
 }
 
 static bool hasThreadBackgroundPriority() {
-#ifdef __APPLE__
+#if defined(__APPLE__) && defined(PRIO_DARWIN_THREAD)
   return getpriority(PRIO_DARWIN_THREAD, 0) == 1;
 #else
   return false;
